propka/lib.py

```diff
@@ -232,9 +232,8 @@ def loadOptions(args):
     Returns:
         argparse namespace
     """
-    # defining a 'usage' message
-    usage = "usage: %prog [options] filename"
-
+    if args == None:
```

Comment:
always use `is` to compare with `None`
```python
    if args is None:
```

Reply:
Or simply pass it through to [parse_args](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.parse_args) which will also handle `None` correctly. I would actually prefer that over the `len(args)` check.

Reply:
Thanks; I've addressed in https://github.com/Electrostatics/propka-3.1/commit/33493cc4d221e4d824bbe4c88cf6a0a3cdc51b4d.  Any concerns about merging with master?

Reply:
I assume you have a reason for removing `propka.cfg` (seems unrelated to the other changes).

The rest looks good to me.

Reply:
Yes, the `propka.cfg` removal (it's a duplicate) is in another pull request to the master branch.  In any case, I'd like to merge.  Thanks.